### PR TITLE
feat(payment): INT-4631 Add Openpay payment method

### PR DESCRIPTION
**Pull request by mauricio-sg**
## What? [INT-4631](https://jira.bigcommerce.com/browse/INT-4631)
Add **Openpay** payment method.

## Why?
To make **Openpay** available at checkout.

## Testing / Proof
Openpay

## Sibling PR
This PR depends on https://github.com/bigcommerce/bigcommerce/pull/42448 to display the **Openpay** logo.
This PR depends on https://github.com/bigcommerce/checkout-sdk-js/pull/1237 to pay, but shouldn't be merged first necessarily.

@bigcommerce/apex-integrations  @bigcommerce/checkout

### CODE REVIEW

File: src/app/locale/translations/en.json [hunk 1]
```diff
@@ -228,6 +228,7 @@
             "klarna_continue_action": "Continue with Klarna",
             "klarna_name_text": "Klarna",
             "masterpass_name_text": "Masterpass",
+            "opy_continue_action": "Continue with openpay",
```
Comment (deov31) [inline src/app/locale/translations/en.json:231]: `Continue with Openpay` 🍹

**Review by deov31 — APPROVED** [review]
LGTM

File: src/app/payment/paymentMethod/PaymentMethodTitle.tsx [hunk 2]
```diff
@@ -105,6 +105,10 @@ function getPaymentMethodTitle(
                 logoUrl: 'https://masterpass.com/dyn/img/acc/global/mp_mark_hor_blk.svg',
                 titleText: '',
             },
+            [PaymentMethodId.Opy]: {
+                logoUrl: cdnPath('/img/payment-providers/opy.png'),
```
Comment (bc-peng) [inline src/app/payment/paymentMethod/PaymentMethodTitle.tsx:109]: Nice job on implementing Openpay. Could I know how to put an icon file on CDN?

Comment (mauricio-sg) [reply src/app/payment/paymentMethod/PaymentMethodTitle.tsx:109]: I don't think it's a CDN as such, but the icon was added as an asset in the PR mentioned in the description above https://github.com/bigcommerce/bigcommerce/pull/42448

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**